tests/integration/actions/collections/base.py

```diff
@@ -3,7 +3,8 @@
 import difflib
 import json
 import os
-import shutil
+
+from distutils.dir_util import copy_tree
```

Comment:
It might be worth keeping `shutil.copytree()` here, because of [this](https://www.python.org/dev/peps/pep-0632/).

Reply:
I did this because copytree doesn't have a exists_true until 3.8 :(

Reply:
added one to common for tests, thx!